ionoscloud/resource_dataplatform_nodepool_test.go

```diff
@@ -0,0 +1,378 @@
+//go:build all || dataplatform
```

Comment:
@cristiGuranIonos same observations I made for the cluster tests are valid here. Maybe we can make another PR in which we will add more tests for these.